cmd/registry.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2017 The kubecfg authors
+//
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ksonnet/ksonnet/metadata"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(regCmd)
+	regCmd.AddCommand(regAddCmd)
+	regCmd.AddCommand(regListCmd)
+}
+
+var regCmd = &cobra.Command{
+	Use:   "registry",
+	Short: `Manage registries for the current ksonnet project`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return fmt.Errorf("Command 'registry' requires a subcommand\n\n%s", cmd.UsageString())
+	},
+}
+
+var regAddCmd = &cobra.Command{
```

Comment:
Getting this unclear error:
```
➜  pr43 git:(pr-43) ✗ ../ks registry add blah
ERROR flag accessed but not defined: name
```

Reply:
^ same

Reply:
Let's remove this command since we don't need it for this afternoon.